pkg/config/config_test.go

```diff
@@ -18,58 +18,58 @@ func TestIsValid(t *testing.T) {
 			name: "correct config with label selector",
 			config: config.WithValues([]*config.Target{
 				config.NewTarget("myproject", []string{"us-east1", "us-west1"}, "team=backend"),
-			}, []int64{5, 30, 60}, 20, nil),
+			}, []int64{5, 30, 60}, 20, 30, nil),
```

Comment:
This test inputs are getting a bit long to exclude keys.

Reply:
I will address this in a different PR. Keeping track at #59